scripts/windows/menu_setup.bat

```diff
@@ -0,0 +1,11 @@
+call deploy_firmware.bat
+call deploy_experimental.bat
+call deploy_contrib.bat
+
+rshell cp /pyboard/lib/contrib/menu.py menu.py
+ren menu.py main.py
+rshell cp main.py /pyboard
+del main.py
```

Comment:
[question] I'm not sure i understand the goal of copying the file twice in order to rename it, so I may be missing something, but i think that the above four lines can all be:

`rshell cp /pyboard/lib/contrib/menu.py /pyboard/main.py`